apps/aot/src/utils/time-utils.ts

```diff
@@ -1,18 +1,25 @@
 export function hasAdventStarted() {
+  const today = new Date().getTime();
   const aoStartTime = new Date(Date.UTC(2024, 11, 1, 5, 0, 0));
-  return Date.now() >= aoStartTime.getTime();
+
+  return today >= aoStartTime.getTime();
 }
 
 export function getCurrentAdventDay() {
   const MS_PER_DAY = 24 * 60 * 60 * 1000;
 
-  const today = Date.now();
+  const today = new Date().getTime();
```

Comment:
does this need to be on EST as well?

Reply:
All these dates are in UTC! 

Also, I am going to change these a bit soon when I migrate everything to be under events/[year]/

getCurrentAdventDay and hasAdventStarted will use the year from the slug and I will use this syntax to make everything clear: example: `new Date(Date.UTC(2024, 11, 1, 5, 0, 0));`